src/components/common/CountDownTimer.vue

```diff
@@ -0,0 +1,65 @@
+<script>
+import VueTimers from 'vue-timers/mixin';
+
+export default {
+  name: 'CountDownTimer',
+
+  props: {
+    timeout: {
+      type: Number,
+      default: 30000,
+    },
+    isLocked: {
+      type: Boolean,
+      default: false,
+    },
+    counter: {
+      type: Number,
+      default: 0,
+    },
+  },
+
+  watch: {
+    isLocked(newValue) {
+      if (!newValue) {
+        return;
```

Comment:
what will if `locked` pass as `false` value and counter will be in working state?
maybe call for `false` value `this.stopRequestTimer()` method?